File: pset6/mario.py
import cs50

def main():
    heigth = checkNumber()
    drow(heigth)
    
def checkNumber():
    while True:
        print ("heigth: ", end="")
        heigth = cs50.get_int()
        if heigth > 0 or heigth <= 23:
            break
    return heigth

def drow(heigth):
    for i in range(heigth):
        for k in range(heigth - i - 1):
            print (" ", end="")
        for j in range(i + 2):
            print ("#", end="")
        print ()
if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int GetHeight ();

int main (void) {
    int height = GetHeight();
    
    for (int i = 0; i < height; i++) {
        for (int j = 2; j <= height - i; j++)
            printf (" ");
        for (int j = 1; j <= i + 2; j++)
            printf ("#");
        printf ("\n");        
    }
}

int GetHeight (void) {
    int h;
    do {
    printf ("height: ");
    h = GetInt();
    if (h == 0) 
        break;
    }
    while (h <= 0 || h > 23);
    return h;
}